app/scripts/services/mapregistry.js

```diff
@@ -36,6 +36,36 @@ angular.module('mapventureApp')
           return registry[uuid] + 'Tour';
         }
         return 'DefaultMapTour';
+      },
+      // Helper function.  Look up a map's UUID from numeric ID,
+      // return Controller name.
+      getControllerNameById: function(id) {
```

Comment:
I think you're right and that we'll want to refactor away from the remaining GeoNode stuff.  Instead of UUIDs we can use slugs or numeric IDs since we're managing them ourselves.  See #434.